plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-role-bindings.yaml

```diff
@@ -476,23 +476,6 @@ items:
   - kind: ServiceAccount
     name: service-account-controller
     namespace: kube-system
-- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Should we leave TODO notes here with the gate name?

Reply:
no, this fixture is auto-generated and is a committed record of the default cluster policy with default feature gate settings